content/applications/general/email_communication/faq.rst

```diff
@@ -1,173 +1,234 @@
-===
-FAQ
-===
+============
+Email issues
+============
 
-This document contains an explanation of the most recurring mailing concerns.
-
-We will start by addressing issues of outgoing emails (ex: my client has not received my email),
-and then, of incoming emails (ex: I do not receive responses from my customers in the database).
+This document contains an explanation of the most recurring emailing issues in Odoo.
 
 Outgoing emails
 ===============
 
 .. _red_envelop:
 
-What do you have to check if your email is not sent?
-----------------------------------------------------
+Email is not sent
+-----------------
 
-The first indicator showing you that the email has not been sent is the red envelope next to the
-date and time of the message.
+The first indicator showing that an email has not been sent is the presence of a :guilabel:`red
+envelope`, next to the date and time of the message, located in the chatter.
```

Comment:
This is optional, but since emojis are used later in the doc I'm suggesting it here. I would also add "_icon_" after the :guilabel: 
```suggestion
The first indicator showing that an email has not been sent is the presence of a red :guilabel:`✉️
(envelope)` icon, next to the date and time of the message, located in the chatter.
```